code/script.js

```diff
@@ -227,99 +229,157 @@ const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
 }
 
+  function findOutCounter() {
+    updateDisplay(++counterValue);
+  }
+
+  function updateDisplay(val) {
+    document.getElementById("counterLabel").innerHTML = val;
+  }
+
+
 // This function to start (and restart) the game
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
-  charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  counterValue = 0;
+  updateDisplay(counterValue);
+  charactersInPlay = CHARACTERS;
+  generateBoard();
+  setSecret();
+  winOrLose.style.display = 'none';
 }
 
 // setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
   const category = questions.options[questions.selectedIndex].parentNode.label
 
-  // This variable stores what option group (category) the question belongs to.
-  // We also need a variable that stores the actual value of the question we've selected.
-  // const value =
+  // This variabel stores the actual value of the question we've selected.
+  const questionValue = questions.value;
 
+  // this variable stores what option group (category) the question belogs to
   currentQuestion = {
     category: category,
-    // value: value
```

Comment:
I like that instead of erasing the value: value you just commented out it to show the difference.